### DISCUSSION TITLE
Phase field modeling

### DISCUSSION THREAD

**Discussion by 412120052** [idaholab/moose#22178] [Q&A Modules: Phase field] [answered]
Hai all,
I have to develop the multiphase field model. I want to extend that model to a polycrystalline one. 
1). I don't have grain growth for one grain i.e. my grain boundaries don't move for one specific grain, but my remaining grains will grow. Is there any example that I can use as an initial step? 
2). how to couple elastic behavior with this 
can you please provide some solutions for this

**Comment by GiudGiud** [comment, marked as answer]
Hello

Please see the documentation for the phase field module. https://mooseframework.inl.gov/modules/phase_field/index.html
The capabilities we have for 1) should be in the grain growth section
For 2), see the elastic driving force for grain growth section section

Guillaume

**Comment by 412120052** [comment]
Thank you for your response
